tests/test_dataset.py

```diff
@@ -225,7 +225,36 @@ def test_label_absences(data_fmodel, inplace, no_sg):
         assert "ABSENT" in result
         assert result["ABSENT"].dtype.name == "bool"
 
+@pytest.mark.parametrize("inplace", [True, False])
+@pytest.mark.parametrize("hkl_index", [True, False])
+def test_remove_absences(inplace, hkl_index):
```

Comment:
This test certainly works, but I think a more thorough test can be done using the `data_fmodel` fixture that tests all sohncke spacegroups with P1/not P1. I don't think its necessary here though given that `label_absences()` is tested in that manner.

Reply:
I'm fine with this one as is